Sources/KituraSampleRouter/RouterCreator.swift

```diff
@@ -88,6 +88,17 @@ public struct RouterCreator {
 
         router.all("/static", middleware: StaticFileServer())
         router.all("/chat", middleware: StaticFileServer(path: "./chat"))
+        
+        router.all("/", middleware: StaticFileServer(path: "./Views"))
+        router.get("/") { request, response, next in
+            response.headers["Content-Type"] = "text/html; charset=utf-8"
+            do {
+                try response.render("home.stencil", context: [String: Any]()).end()
```

Comment:
As far as I can see, this isn't a Stencil file, it's just HTML. Should it just be `home.html` and served via `response.send()`?